DataRepo/models.py

```diff
@@ -38,6 +40,21 @@ class TracerLabeledClass:
     MAX_LABELED_COUNT = 20
 
 
+def atom_count_in_formula(formula, atom):
+    """
+    Return the number of specified atom in the compound.
+    Returns None if atom is not a recognized symbol
+    Returns 0 if the atom is recognized, but not found in the compound
+    """
+    substance = Substance.from_formula(formula)
+    count = None
+    if atom in symbols:
+        # composition returns dict of {atomic_weight: count}
+        # symbols is a tuple of elements in atomic weight order
+        count = substance.composition.get(symbols.index(atom) + 1, 0)
```

Comment:
B. I'd prefer you use the `atomic_number` method from inside `periodic.py` instead of `symbols.index(atom) + 1`. It handles an exception for case.  It should be more robust if they ever change things.

Reply:
I was looking for a better way to do that, I'll see if I can find that method and use it, thanks!

Reply:
Hmmm. I don't see a way to go from `symbol` to `atomic_number` without directly accessing the `symbols` list. [`atomic_number`](https://bjodah.github.io/chempy/latest/_modules/chempy/util/periodic.html#atomic_number) takes a `name` and returns a `number`. For now, I think I'll have to keep it this way.

Reply:
I thought all you would have to do is import the atomic_number method from periodic.py:
```
def atomic_number(name):
    try:
        return symbols.index(name) + 1
    except ValueError:
        return lower_names.index(name.lower()) + 1
```
And call it, passing the atom name, e.g.:
```
        count = substance.composition.get(atomic_number(atom), 0)
```
What am I missing?  Can the method not be imported for some reason?

Reply:
The `atom_count` method takes a `symbol`. The `atomic_number` function takes a `name`. I see no way to convert between `name` and `symbol` (or `symbol` and `name`) without accessing the `symbols` list directly.

Reply:
Huh.  Well, I assume you know what you're talking about and will leave it up to you, but when I look at the tests, I see atom supplied as a string: `alanine.atom_count("C")`. That string is passed into this surrounding method with `atom_count_in_formula(self.formula, atom)`. I wouldn't understand why you counldn't just import this method:

https://github.com/bjodah/chempy/blob/dd8b934e9f2732d26ba6b5a71e5ee908079bd739/chempy/util/periodic.py#L257

And call it like this:
```suggestion
        count = substance.composition.get(atomic_number(atom), 0)
```
Wouldn't that look up the atomic number of the symbol here?:

https://github.com/bjodah/chempy/blob/dd8b934e9f2732d26ba6b5a71e5ee908079bd739/chempy/util/periodic.py#L9

Reply:
Oh wait - I see what the confusion is.  The name of the argument for `atomic_number` is `name` - **BUT** it is doing a lookup in the `symbols` array(?).  And if that gets an exception, it tries to look it up as a name.  I'll admit that's weird.  I hadn't noticed that at first.  But - it should work.

Reply:
Yup, you're right. Should work. That's a terrible argument name... 🤦 

I'll make that change.